milo-rastgoo.txt

```diff
@@ -0,0 +1,103 @@
+Link to codepen assignment: https://codepen.io/rastgoo/pen/ZEEXWvG
+
+1. What is Semantic HTML? 
+semantic html introduces elements with meaning to web page layouts. it is used both for web browsers and user agents such as screen readers.
+2. What is HTML used for? 
+HTML is used to markup contents of a web page. Markup part is similar to traditional editorial tags/markings for newspapers/books editorials and Hypertext part of it refers to webpages with innerlinks to each other and other resources. 
+3. What is an attribute and where do we put it?
+Attributes provide additional information about an element's configuation or behaviour.An attribute is added to an HTML start tag. 
+4. What is the h1 tag used for? How many times should I use it on a page?
+h1 is the first level heading. General best practice is to use it only once on every webpage for the most important heading.
+5. Name two tags that have required attributes
+a, img
+6. What do we put in the head of our HTML document? 
+meta information that is targetted to web browsers, search engines etc. and not for human beings.
+7. What is an id? 
+id is an identifier attribute which is used uniquely in a webpage. It is used to point to an HTML tag to be used in CSS and/or javascript.
+8. What elements can I add an id to? 
+according to w3schools, In HTML 5 the id attribute can be used on any HTML tag even if not useful. In previous version there are some exclusions and they cannot start with numbers.
+9. How many times can I use the same id on a page? 
+an id is a unique pointer and an id=value pair should be used only once on a webpage.
+10. What is a class? 
+class is an HTML attribute that specifies one or more classes for a given html element. classes can be used by css and/or javascript to make changes to an html element.
+11. What elements can I add a class to? 
+As of HTML 5, all HTML elements can have class attribute eventhough it might not be useful.
+12. How many times can I use the same class on a page? 
+AFAIK, there are no limits on using the same class on a page.
+13. How do I get my link to open in a new tab?
+using target attribute in a tag: target="_blank"
+14. What is the alt attribute used for? 
+alt attribute is used for images where the image cannot be read from source, for screen readers and accessiblity cases, search engine optimization and image search. etc.
+15. How do I reference an id?
+in CSS we can refer to ids using #idName
+16. What is the difference between a section and a div
+A div is a presentational HTML tag to group pieces of content together. section is a semantic HTML tag that specifies and seperates a section of a page from the rest of the page content.
+17. What is CSS used for? 
+CSS as Cascadig Style Sheets is used to define the presentation and style of an HTML document. (according to Mozilla its also used to describe elements rendering on screen, paper, in speech, or other media??!!, I'll research this. very interesting)
```

Comment:
Great work. Re: Question 17 maybe you can share what you found with the team? : )

Reply:
Will do.